patterns/project-layout.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Title: Project Layout
+ * Slug: twentytwentyfour/project-layout
+ * Categories: text
+ */
+
+?>
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"6vw","right":"6vw","bottom":"6vw","left":"6vw"}},"elements":{"link":{"color":{"text":"#fffffffc"}}},"color":{"background":"#181818","text":"#fffffffc"}},"layout":{"type":"constrained"}} -->
```

Comment:
This should have wide width. The colors should use the presets from the theme instead of the hex values and the spacing should also try to use the presets too

Reply:
Thanks for all your help. I not founded the preset color, so i thought wasn't so important inside the all theme (i'll fix it).
How can i achive to use the preset for the spacing if not using the block spacing? 

Again, thanks for all your help

Reply:
This is where you use the presets:

<img width="318" alt="Screenshot 2023-09-04 at 11 55 50" src="https://github.com/WordPress/twentytwentyfour/assets/3593343/e8cb9de2-dd68-415e-a88b-179accd3f283">

Reply:
I think now is ok. Only a little question. There's no the same color of the background (based on figma file) i used the contrast color preset, is ok or i have to create the preset myself into the theme.json?)

Sorry for all the extra work i make you do...i'm trying to learn fastest possible for make no errors.

Reply:
it's absolutely no issue! and thank you for following up on our reviews, that's what we are here for. You shouldn't need to create any presets, but chose what looks closer to the design. We have iterated away from some of the color to make them more accesible.

Reply:
Thank u very much for the kind words.
Ok, i choosed the contrast color preset, the #222222 hope is ok for the pattern.